metals/src/main/scala/scala/meta/internal/worksheets/WorksheetProvider.scala

```diff
@@ -313,9 +313,9 @@ class WorksheetProvider(
   private def getMdoc(target: BuildTargetIdentifier): Option[Mdoc] = {
 
     def isSupportedScala3Version(scalaVersion: String) = {
-      // Worksheet support for Scala 3 started with 0.26.0-RC1
+      // Worksheet support for Scala 3 started is working for 3.0.0-M1
```

Comment:
I'm not sure I understand this comment do you mean?
```suggestion
      // Worksheet support for Scala 3 starts with 3.0.0-M1
```
Or something along that lines? Does this also mean that someone on the deprecated 0.27.0 won't have worksheets? I'm fine if that's the case, just wondering.

Reply:
Ach, my brain doesn't seem to be working properly anymore, I will fix the comment!